packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -7,13 +7,13 @@
                 <button
```

Comment:
Should we be using f-button now seen as we're in this component.

Reply:
f-button had specifc styling associated that we wanted to avoid here, I believe.

Reply:
Happy to address this in a later PR if we think it's worth updating.
It feels like the right solution is to have a button variant with the styling we want, rather than avoiding using f-button because of styling.